fabric-structure-api-v1/src/main/resources/fabric-structure-api-v1.accesswidener

```diff
@@ -0,0 +1,3 @@
+accessWidener v1 named
+accessible class net/minecraft/util/registry/DynamicRegistryManager$Info
+mutable    field    net/minecraft/structure/pool/StructurePool    elementCounts    Ljava/util/List;
```

Comment:
No need for this if you mark your `@Accessor` setter with `@Mutable` in `StructurePoolAccessor`